web/src/features/moreCast2/components/DataGridColumns.tsx

```diff
@@ -95,15 +95,9 @@ export const columnGroupingModel: GridColumnGroupingModel = [
 // and visibility of our weather parameter tabs
 function columnGroupingModelChildGenerator(weatherParam: string) {
   // For a given weather model, there are tabs present in the datagrid for each WeatherDetermiante except
-  // WeatherDeterminate.NULL. Additioanlly, bias adjusted predictions only exist for temp and rh.
+  // WeatherDeterminate.NULL
   let determinates: WeatherDeterminate[] = []
```

Comment:
I haven't stepped through the code, but I'm not sure this change will be valid if we're not showing the precip and wind direction bias adjusted values in the data grid?

Reply:
Yep it still works